Doc/library/shutil.rst

```diff
@@ -532,11 +532,13 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
 
    *root_dir* is a directory that will be the root directory of the
    archive; for example, we typically chdir into *root_dir* before creating the
-   archive.
+   archive.  This also means that all paths in the archive will be relative to
+   *root_dir*.
```

Comment:
I think this may be better as 

```
*root_dir* is a directory that will be the root directory of the
archive, all paths in the archive will be relative to it; for example, 
we typically chdir into *root_dir* before creating the archive.
```

Reply:
I like it better. I can remember having some difficulties in the wording of this. Thanks for your suggestion.